pyproject.toml

```diff
@@ -8,6 +8,7 @@ dependencies = [
     "langchain-core",
     "langchain-community",
     "langchain_openai",
+    "langchain-google-genai",
```

Comment:
what do you think of making this an optional dependency?
something like `ragas[google]`.

I'm waiting for our approval to use GCP so will test these after that